back-end/server.js

```diff
@@ -3,8 +3,8 @@ import cors from "cors";
 import transcripts from "./api/routes/transcripts.route.js";
 import projects from "./api/routes/projects.routes.js";
 
+// Set-up for the express app/store with route endpoints for transcripts and projects 
 
-// This is the express config file
 const app = express()
 
 app.use(cors())
```

Comment:
do we still need cors?

Reply:
We would need cors when running our app without deploying it because cors resolves issues with communication in-between two localhost ports.

Reply:
Got it!